Resources/Prototypes/Loadouts/Jobs/Civilian/passenger.yml

```
@@ -112,3 +112,88 @@
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitMilitaryColorLightBrown
+
+- type: loadout
+  id: BrownMJ
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitMilitaryColorBrown
+
+- type: loadout
+  id: MaroonMJ
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitMilitaryColorMaroon 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)